code/modules/mob/living/carbon/human/species_types/mushpeople.dm

```diff
@@ -0,0 +1,46 @@
+/datum/species/mush //mush mush codecuck
+	name = "Mushroomperson"
+	id = "mush"
+	mutant_bodyparts = list("caps", "tail_human", "ears", "wings")
+	default_features = list("caps" = "Round", "tail_human" = "None", "ears" = "None", "wings" = "None")
+
+	fixed_mut_color = "DBBF92"
+	hair_color = "FF4B19" //cap color, spot color uses eye color
+	nojumpsuit = 1
+
+	say_mod = "poofs" //what does a mushroom sound like
+	species_traits = list(MUTCOLORS, NOEYES, NO_UNDERWEAR, NOBREATH)
+	speedmod = 1.5 //faster than golems but not by much
+
+	punchdamagelow = 6
+	punchdamagehigh = 14
+	punchstunthreshold = 14 //about 44% chance to stun
```

Comment:
thats pretty fuckin strong dude

Reply:
it's what bman had, i could have the species punch way slower but have stronger fists if that would help

Reply:
Is this a dark souls reference?

Reply:
actually genuinely yes

Reply:
i'll just make a really quick martial art that does mjolnir punches but every punch they make requires a time waiting and that will satisfy balance and my eternal love for the shroom dudes in dark souls 1

Reply:
In my opinion, this is way too strong.

Reply:
they get up fast enough to make it only lethal if the victim is right up against a wall, which i don't find to be too problematic